subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1248,6 +1248,28 @@ static int prop_get(struct net_buf_simple *buf,
 	return sensor_ch_encode(buf, format, &sensor_val);
 }
 
+/* If light is already in `state`, or transitioning to `state`,
+ * update the target level to new value.
+ */
+static void update_lightness_setpoint(struct bt_mesh_light_ctrl_srv *srv,
```

Comment:
Is there a reason why this is defined twice? As far as I can see the definitions are identical and can be put outside any #ifdefs

Reply:
Thought it'd be nice to have it close to `prop_set` where it's used and the duplicate will be removed once the deprecated sensor API is removed. 
Can move it above the first `prop_get` if you insist?

Reply:
There is a _lot_ of rearranging functions as part of the removal of the old sensor API anyways, I suggest to move it to right before the #ifndef above prop_get, to not look confusing, especially if this goes into NCS 2.7.0 (which it might now if this becomes ready to merge today)